src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -1,28 +1,34 @@
 <template>
-  <div class="ec-main-container ec-blog-item">
-    <div class="tw-grid tw-border-b tw-border-solid tw-border-gray-6">
+  <transition-group
+    class="ec-main-container ec-blog-item"
+    tag="div"
+  >
+    <div
+      v-for="post of posts"
+      :key="post.id"
+      class="tw-grid tw-border-b tw-border-solid tw-border-gray-6"
+      data-test="ec-blogItem__post"
+    >
       <div class="tw-col-2">
-        <img class="tw-w-96 tw-h-96 tw-rounded" :src="featuredImage">
+        <img
+          class="tw-w-96 tw-h-96 tw-rounded"
+          :src="post.featuredImage"
```

Comment:
don't read from post, pass and access only `featuredImage`

https://en.wikipedia.org/wiki/Law_of_Demeter